docker/DockerFile.amd64-build

```diff
@@ -3,31 +3,51 @@
 
 FROM --platform=linux/amd64 eclipse-temurin:21
 
-# To enable running android tools such as aapt
-RUN apt-get update && apt-get -y upgrade
-RUN apt-get install -y libz1 libstdc++6 unzip zip
-# For Go:
-RUN apt-get -y --no-install-recommends install curl gcc
-RUN apt-get -y --no-install-recommends install ca-certificates libc6-dev git
-
-RUN apt-get -y install make
-
-RUN mkdir -p build
 ENV HOME /build
-
-# Make android sdk location, the later make step will populate it.
-RUN mkdir android-sdk
 ENV ANDROID_HOME $HOME/android-sdk
 ENV ANDROID_SDK_ROOT $ANDROID_HOME
 ENV PATH $PATH:$HOME/bin:$ANDROID_HOME/platform-tools
 
+RUN mkdir -p \
```

Comment:
(mild suggestion, no requirement to act on it; can also do this in a followup change)

When Dockerfile RUN statements get long enough to need copious newline escaping, I tend to extract the build steps into a build.sh or similar, and compact the Dockerfile to copying that build script into the image and running it. That makes editors happier to edit the build script, and also makes it visible to tools like shellcheck (though I don't think we have it set up in this repo).